components/Aduan/Milestone.vue

```diff
@@ -0,0 +1,212 @@
+<template>
+  <ol class="relative border-l border-gray-200 ml-4">
+    <li
+      v-for="milestone in dataMilestone"
+      :key="milestone.status"
+      class="mb-3 ml-6"
+    >
+      <span
+        class="absolute flex items-center justify-center w-[26px] h-[26px] bg-gray-50 rounded-lg -left-3 ring-8 ring-white"
+      >
+        <BaseIconSvg
+          :icon="`/icon/${getStatusTextAndIcon(milestone.status).icon}`"
+          class="!shadow-lg !w-[14px] !h-[14px]"
+          :fill-color="getStatusTextAndIcon(milestone.status).fillColor"
+        />
+      </span>
+
+      <CardMilestone>
+        <TextMilestone>
+          <template #date-time>
+            {{ milestone.date }} - {{ milestone.time }}
+          </template>
+
+          <template #information>
+            {{ getStatusTextAndIcon(milestone.status).conditionAduan }}
+          </template>
+
+          <template #status>
+            <b>{{ getStatusTextAndIcon(milestone.status).status }}</b>
+            <span class="text-gray-800">oleh</span>
+            <b>{{ milestone.name }}</b>
+          </template>
+        </TextMilestone>
+
+        <template
+          v-if="
+            milestone.status === dataStatusMilestone.followUp.status ||
+              milestone.status === dataStatusMilestone.done.status
+          "
```

Comment:
minor

maybe we can use some method here to make the condition more maintainable and readable

```js
methods: {
  isFollowUpOrDone(milestone, dataStatusMilestone) {
    return milestone.status === dataStatusMilestone.followUp.status
      || milestone.status === dataStatusMilestone.done.status
  }
}
```

Reply:
i am change [here](https://github.com/jabardigitalservice/super-app-web/pull/42/commits/51159d5a995df1142202a50cb7d861b14a32534a)